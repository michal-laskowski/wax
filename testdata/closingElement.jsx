export function closingElement() {
    return (
        <>
            <area />
            <area></area>
            <base />
            <br />
            <col />
            <command />
            <embed />
            <hr />
            <hr></hr>
            <img />
            <input />
            <keygen />
            <link />
            <meta />
            <param />
            <source />
            <track />
            <wbr />
            {/* Not void elements */}
            <div />
            <span />
            <p />
            <custom-element></custom-element>
            <custom-element />
        </>
    );
}