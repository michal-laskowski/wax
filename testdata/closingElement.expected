<area>
<area>
<base>
<br>
<col>
<command>
<embed>
<hr>
<hr>
<img>
<input>
<keygen>
<link>
<meta>
<param>
<source>
<track>
<wbr>

<div></div>
<span></span>
<p></p>
<custom-element></custom-element>
<custom-element></custom-element>